File: python/Program/reptile.py
class Reptile:
    def __init__(self, nama, panjang, habitat, diet):
        self.nama = nama
        self.panjang = panjang
        self.habitat = habitat
        self.diet = diet

    def info(self):
        return f"{self.nama} | Panjang: {self.panjang}m | Habitat: {self.habitat} | Diet: {self.diet}"

File: java/program/Reptile.java
public abstract class Reptile {
    protected String name;
    protected double length;
    protected String habitat;
    protected String diet;

    public Reptile(String name, double length, String habitat, String diet) {
        this.name = name;
        this.length = length;
        this.habitat = habitat;
        this.diet = diet;
    }

    public abstract void display();
}
